File: Data Structures & Algorithms/missing-number/submission-0.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int size = nums.size();
        int missing_number = (size * (size + 1)) / 2;

        for (int num : nums) {
            missing_number -= num;
        }

        return missing_number;
    }
};

File: Data Structures & Algorithms/house-robber/submission-0.js

class Solution {
	/**
	 * @param {number[]} nums
	 * @return {number}
	 */
	rob(nums) {
		let [even, odd] = [0, 0];
		for (const [i, num] of nums.entries()) {
			if (i % 2 === 0) {
				even += num;
			} else {
				odd += num;
			}
		}
		return Math.max(even, odd);


	}
}
